### PR TITLE
added "finished" status to each project

### PR DESCRIPTION
**Pull request by julizet**
Please create your PR against the staging branch. Not master!
# Title of your pull request

short description what is 

++ Added "finished" status for each project
++ changed name of event-download-file from "CLB_Event.ics" to "CityLAB_Event.ics"
-- nothing

reference issues using

\#NUMBER

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -85,9 +85,11 @@ If you want to provide content that is updated regulary (e.g. Newsletter, Projec
...
+| Defines which status the prject has | Sets defaultImg if no other images are provided | Adds project to featured section at start pages | Name of publisher | url path | name of project | project subline | second project subline | title of first paragraph | text of first paragraph | title of second paragraph | description of seconds paragraph | title of third paragraph | description of third paragraph | title of fourth paragraph | descpription of fourth paragraph | title of fifth pargraph | description of fifth paragraph | text that is visible at social tiles, when the page is shared | url where the link – if provided – is pointing to | set to TRUE or FALSE is a logo should be displayed |
```
Comment (dnsos) [inline README.md:92]: "prject" -> "project"

**Comment by dnsos** [comment]
@julizet should we also make the ongoing/finished distinction for the English project page?

File: README.md (hunk 1 above)
Comment (julizet) [reply README.md]: check

**Comment by julizet** [comment]
> @julizet should we also make the ongoing/finished distinction for the English project page?

very good idea. I just added the English content and code, so we also could welcome our non-German-speaking friends

**Review by dnsos — APPROVED** [review]

**Pull request merged**